cppcache/src/TcrMessage.hpp

```diff
@@ -50,12 +50,15 @@
 namespace apache {
 namespace geode {
 namespace client {
+
 class TcrMessage;
 class ThinClientRegion;
 class ThinClientBaseDM;
 class TcrMessageHelper;
 class TcrConnection;
 class TcrMessagePing;
+enum class CqState;
```

Comment:
Why does this need to be forward declared?

Reply:
This header does include, nor does it need to include the enum. I suppose it wouldn't hurt, but it is one less include.